man/systemd-system.conf.xml

```diff
@@ -171,6 +171,126 @@
         effect if a hardware watchdog is not available.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>IOCostQOSPath=</varname></term>
+        <listitem><para> Configure the absolute path to device to be configured
+        in io.cost.qos file of the root cgroup.</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>IOCostQOSCtrl=</varname></term>
+        <listitem><para>Configure whether the io.cost.qos parameters are
+        controlled by the kernel or by the user. Allowed values are 'auto' and 'user'.
+        <varname>IOCostQOSRPct</varname>, <varname>IOCostQOSRLat</varname>, <varname>IOCostQOSWPct</varname>,
+        <varname>IOCostQOSRLat</varname>, <varname>IOCostQOSWPct</varname>, <varname>IOCostQOSWLat</varname>
+        <varname>IOCostQOSMin</varname>, and <varname>IOCostQOSMax</varname> are all ignored if value is set to 'auto'. </para></listitem>
```

Comment:
this option should not be exposed. Instead, if any of the other values are set we should set this to "user" implicitly, and otherwise to "auto".